src/components/HomePage.tsx

```diff
@@ -1,103 +1,38 @@
 import React from 'react';
```

Comment:
This component is HomePage.tsx and you're using UserStyle here (incorrect)
You should create Home styles separately